### ISSUE TITLE
cancel sort in remote order

### ISSUE THREAD

**Issue by abonec** [AllenFang/react-bootstrap-table#1282]
I use table sort only for display sort order because of that I use remote option. But table don't have any option to cancel sorting. I try to use `options` but it seems to ignore that.

My code snippet is https://gist.github.com/abonec/3c59009ca365259ddd9dcc32f21cab96

**Comment by AllenFang** [comment]
@abonec, sorry I am not very understand about what's means for cancel sorting, you mean the sort direction is still remain on header and data seems like no change?

**Comment by abonec** [comment]
@AllenFang I use table only to visualize data and sorting indicators and callbacks that gives me information about sorting that user choose. This information I send to the server and server send me back already sorted data. Table only visualize sort order in headers and actually don't need to sort it.

In term `cancel sorting` I mean to reset triangles in headers which visualize sort order.

**Comment by AllenFang** [comment]
umm.... so did you want a feature which can help you to remove the triangles on header???? and when did you want it, what user action will trigger this `cancel sorting`???

**Comment by abonec** [comment]
@AllenFang usually (i.e. DB systems) after the third click on the header will reset sort (first two will sort in acs/desc and third will reset). It will be great.
But I asked you for API, may be about callback.

**Comment by AllenFang** [comment]
> after the third click on the header will reset sort (first two will sort in acs/desc and third will reset). It will be great.

ok, got it. I will support it, but actually need some time, it's minor feature so without any schedule promise

> But I asked you for API, may be about callback.

you means you want to have a callback or something to know what column and order that user trigger?

**Comment by abonec** [comment]
> you means you want to have a callback or something to know what column and order that user trigger?
no, I mean to set some props to the table for refresh order. If you look at the source code above I set `sortOrder` and `sortName` when `cancelOrder` button was pressed but this has no affect in remote mode.

**Comment by AllenFang** [comment]
understand, thanks you, so let me check again, there're two major thing you were hope:
1. Can cancel the sorting via clicking, ex: --click--> DESC --click--> ASC --click-->NONE
2. you hope there's a function or props can cancel the sorting.

**Comment by abonec** [comment]
exactly

**Comment by AllenFang** [comment]
I'll support it, but need some time

**Comment by AllenFang** [comment]
> you hope there's a function or props can cancel the sorting.

Noe, It support on `v3.3.4`, check [this](https://github.com/AllenFang/react-bootstrap-table/blob/master/examples/js/sort/clean-sorted-table.js#L24-L26) to see how to clean a sorted table.



> Can cancel the sorting via clicking, ex: --click--> DESC --click--> ASC --click-->NONE

Actually, it did need some time, no schedule, sorry.

**Comment by abonec** [comment]
it works, thx

**Comment by somnathpathak** [comment]
Does `v4.3.1` supoort the below functionality now?

> Can cancel the sorting via clicking, ex: --click--> DESC --click--> ASC --click-->NONE

I need to use something like this on my table.

**Comment by spal0002** [comment]
Hi i am using react-bootstrap-table ^4.3.1
I want remote pagination so i set remote={true} but i want local sorting.

Can you please help..
